virttest/test_setup.py

```diff
@@ -2034,9 +2037,12 @@ def cleanup(self):
             if os.path.exists(self.libvirtd_backup_path):
                 os.rename(self.libvirtd_backup_path, self.libvirtd_path)
             if self.user.count('EXAMPLE'):
-                logging.debug("Delete the created user 'testacl'.")
-                cmd = "userdel -r testacl"
+                self.user = 'testacl'
+            if self.params.get('add_polkit_user'):
```

Comment:
still we would not require `add_polkit_user`, we can do a similar check that we perform before creating the user,
```python
if self.user and if process.system("id %s" % self.user, ignore_status=True) == 0:
    # delete the user
```

Reply:
@balamuruhans Thanks, but I don't want to delete the exited user in the test env, it looks like need a flag to save the user is created by scripts or not

Reply:
@hsmj1412, even if anyother existing user are there in test env, self.user will be set only by the script right ? then how does it affect other user ? IMHO `add_polkit_user` is not necessary.

Reply:
@balamuruhans I mean some people run this script in their workstation, and they have created and used this user(testacl), so we should not delete it.